dictdiffer/utils.py

```diff
@@ -11,10 +11,14 @@
 
 import sys
 
-from ._compat import izip_longest, num_types, string_types
+from ._compat import (MutableMapping, MutableSequence, izip_longest, num_types,
+                      string_types)
 
 EPSILON = sys.float_info.epsilon
 
+DICT_TYPES = (MutableMapping, )
```

Comment:
I feel like this is duplicated knowlege as it is present [here](https://github.com/inspirehep/dictdiffer/pull/1/files#diff-d8d1353909cc9f4e93181a1e680a2ae9R35) as well. Maybe we should move DICT_TYPES and LIST_TYPES in a centralised place and import from there?

Reply:
good point but we cannot do it, ``__init__`` imports from ``utils``